lib/public/js/chart.js

```diff
@@ -153,3 +144,5 @@ const toggleColor = () => {
     btn.classList.toggle('active');
     btn.classList.toggle('inactive');
 };
+
+(typeof module !== 'undefined' && module !== null ? module : {}).exports = { chart, toggleColor, saveChart, sum, slugify };
```

Comment:
Do we actually need this checking? Can't we just do `module.exports`?

Reply:
@wildan3105 module.exports gives an error in the console `module is not defined`. I'm still not sure about the reason. It could be because we are trying to export a module which gets compiled after the html files. but in the html files, we call these functions too. I found a workaround for checking and considering it only when the js files starts compiling. I'm happy to rewrite the code if it can be resolved  in another way.

Reply:
I see got it. I think it’s because of our eslint rule. Can leave it that way then 👌